packages/atomic/src/components/atomic-did-you-mean/atomic-did-you-mean.tsx

```diff
@@ -10,9 +10,9 @@ import {
 import {sanitize} from '../../utils/xss-utils';
 
 /**
- * The DidYouMean component is responsible for handling query corrections.
- * When a query returns no result but finds a possible query correction, the component either suggests the correction or
- * automatically triggers a new query with the suggested term.
+ * The `atomic-did-you-mean component` is responsible for handling query corrections.
```

Comment:
should the "`" be before the word component?

Reply:
The humour in this typo is not lost on me! That is correct, thanks.